netkan/netkan/metadata.py

```diff
@@ -83,6 +84,7 @@ def __init__(self, filename=None, contents=None):
         else:
             self.contents = contents
         self._raw = json.loads(self.contents)
+        self.version = self.Version(self._raw.get('version'))
```

Comment:
I this supposed to be
```python
self.version = Version(self._raw.get('version'))
```

Reply:
`Version` is defined within the body of `Ckan`, so this is shorthand for `Ckan.Version`, kind of like a class constant, I thought.

Reply:
Oh so it is, missed the indenting in GitHub. That's an interesting approach, makes sense that is closely tied to Ckan. But if we ever want to use it outside the Ckan class we might want to make it a top level.

Reply:
You can still use it like this: `some_version = Ckan.Version('somestring')`, without instantiating a `Ckan` object, like I did in the tests.
But yes, the option is there to move it out of `Ckan` if there is a need for it.

Reply:
You can, and that's fine for testing. But I think we would make it a top level class if we needed it beyond that.